data/card_catalogue/unique-1.json

```diff
@@ -154,8 +186,11 @@
     "upgrade": "",
     "tier": "blue",
     "details": {
-      "rules": "Learn the current location of an item you've touched in the past.",
-      "flavour": "\"At last I'll be able to track down the missing socks\" - Cordelia Hex, Angry Witch",
+      "rules": {
+        "active": "Learn the current location of an item you've touched in the past.",
+        "passive": ""
+      },
+      "flavour": "At last, the location of the missing socks.",
```

Comment:
Cutting out direct references to 'Witch' and 'Wizard' so we can be free to canonize whatever we want on a campaign-by-campaign basis. Sorry Cordelia!
In a similar vein I've also avoided using 'summon' unless the card might actually summon something in Tabletop Simulator, and opted for 'conjure' and 'evoke' situationally. This didn't affect many cards.